features/step_definitions/broker_employee_quote_steps.rb

```diff
@@ -337,3 +337,49 @@ def broker_agency(*traits)
 When(/^the broker clicks Dental Features$/) do
   find('.interaction-click-control-dental-features-and-cost-criteria').click
 end
+
+And(/^prospect employer exist$/) do
+  find(BrokerHomePage.employers_tab).click
+  find(BrokerEmployersPage.add_prospect_employer_btn).click
+  fill_in BrokerAddProspectEmployerPage.legal_name, with: 'emp1'
+  fill_in BrokerAddProspectEmployerPage.dba, with: '101010'
+  find(BrokerAddProspectEmployerPage.entity_kind_dropdown).click
+  find(BrokerAddProspectEmployerPage.select_c_corporation).click
+  fill_in BrokerAddProspectEmployerPage.address_1, with: '1818 exp st'
+  fill_in BrokerAddProspectEmployerPage.city, with: 'Washington'
+  fill_in BrokerAddProspectEmployerPage.zip, with: '20002'
+  fill_in BrokerAddProspectEmployerPage.area_code, with: '202'
+  fill_in BrokerAddProspectEmployerPage.number, with: '5551212'
+  find(BrokerAddProspectEmployerPage.confirm_btn).click
```

Comment:
This is a bit an anti-pattern. A `Given` step should never really use the browser other than `visit`. Instead of creating the existing employer through the browser let's use a helper method to establish an employer with a roster and an existing quote. By avoiding the browser it will be faster and we're isolating the tests to only what is supposed to be tested.